treehouse/pyspark.py

```diff
@@ -0,0 +1,187 @@
+from pyspark.sql import SparkSession, DataFrame
+from pyspark.sql.types import (
+    ArrayType,
+    BooleanType,
+    StructType,
+    StructField,
+    StringType,
+    IntegerType,
+    TimestampType,
+    DateType,
+    FloatType,
+)
+from pyspark.sql.functions import lit, col
+import re
+
+
+def create_spark_session(
+    appName: str, master: str = "yarn", enable_bigquery: bool = False
```

Comment:
seeing mixed casing between arguments, prefer snake_case myself but at least has to be consistent